docs/documentation/development/bindings/discovery-services.md

```diff
@@ -100,6 +100,12 @@ The following example implementation for `startScan` is taken from the `HueLight
     }
 ```
 
+### Re-Discovered Results and Things
+
+The `getThingUID` method of the discovery service should create a consistent UID every time the same thing gets discovered.
```

Comment:
I would at least capitalize or maybe even emphasize `Thing` throughout the text.

Reply:
There are a lot of other references to "thing" on the site. I would like to leave it like this for consistency reasons.